src/sentry/migrations/0397_backfill_user_avatars.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.2.28 on 2023-01-31 20:37
+
+from django.db import migrations
+
+from sentry.api.utils import generate_region_url
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_user_avatar(apps, schema_editor):
+    User = apps.get_model("sentry", "User")
+    # UserAvatar = apps.get_model("sentry", "UserAvagtar")
+
+    for user in RangeQuerySetWrapperWithProgressBar(User.objects.all()):
+        avatar = user.avatar.first()
+        if avatar is None:
+            continue
+        user.avatar_type = avatar.avatar_type
+        if avatar.avatar_type == 1:
+            user.avatar_url = f"{generate_region_url()}/avatar/{avatar.ident}/"
+        user.save()
```

Comment:
Also would be slightly better to use `update_fields` to only save the updated `avatar_url` field